src/Main.java

```diff
@@ -1,8 +1,8 @@
-import java.util.Arrays;
-
 public class Main {
+    private static final Employee[] employees = new Employee[10];
```

Comment:
поля static и final надо называть в верхнем регистре
```suggestion
    private static final Employee[] EMPLOYEES = new Employee[10];
```